src/components/home/modals/ColdSignature.vue

```diff
@@ -125,13 +125,13 @@ export default {
             var text = ''
             if (this.qrTotalPage === 1) {
                 let data = JSON.parse(JSON.stringify(this.dataObject))
-                if (data.hasOwnProperty('contractId')) {
+                if (data.opc === OPC_FUNCTION) {
                     delete data.senderPublicKey
                 }
                 text = JSON.stringify(data).replace(/"amount":"(\d+)"/g, '"amount":$1')
             } else {
                 let tempData = JSON.parse(JSON.stringify(this.dataObject))
-                if (tempData.opc === 'contract') {
+                if (tempData.opc === OPC_CONTRACT) {
```

Comment:
why not `use this.dataObject.transactionType`?

Reply:
In the dataObject, there is no such variable，only have opc.